variables.sh

```diff
@@ -14,6 +14,7 @@ GRPC_WEB_VERSION=${GRPC_WEB_VERSION:-1.3.1}
 GRPC_GATEWAY_VERSION=${GRPC_GATEWAY_VERSION:-v2.0.1}
 UBER_PROTOTOOL_VERSION=${UBER_PROTOTOOL_VERSION:-1.10.0}
 SCALA_PB_VERSION=${SCALA_PB_VERSION:-0.11.0}
+MYPY_VERSION=${MYPY_VERSION:-3.3.0}
```

Comment:
we leverage [Renovate](https://github.com/namely/docker-protoc/blob/master/.github/renovate.md#how-to-keep-maintaining-renovate-in-this-repo) to manage dependencies.
Would you mind adding the configuration to https://github.com/namely/docker-protoc/blob/master/.github/renovate.json5 ?

I think it should be something like this (I'll try to test it later):
```
  {
    fileMatch: [
      "variables.sh$"
    ],
    matchStrings: [
      "MYPY_VERSION =\\$\\{MYPY_VERSION:-(?<currentValue>.*?)\\}\\n"
    ],
    depNameTemplate: "mypy-protobuf",
    datasourceTemplate: "pypi",
  }
```